### PR TITLE
Feature/bosh sf

### PR DESCRIPTION
**Pull request by muralisc**
Spec file changes for handling multiple directors.

### CODE REVIEW

File: config/final.yml [hunk 1]
```diff
@@ -2,6 +2,5 @@
 blobstore:
   provider: s3
   options:
-    bucket_name: sf-boshrelease
-    folder: service-fabrik-release-blobs
+    bucket_name: sf-boshrelease/service-fabrik-release-blobs
```
Comment (kamath-prasad) [inline config/final.yml]: why is this change needed ?

File: jobs/service-fabrik-broker/spec [hunk 2]
```diff
@@ -188,32 +188,8 @@ properties:
...
-  broker.bootstrap_bosh_director.username:
-    description: "Admin username for the Boot Strap Bosh director"
-  broker.bootstrap_bosh_director.password:
-    description: "Admin password for the Boot Strap Bosh director"
-  broker.bootstrap_bosh_director.skip_ssl_validation:
-    description: "Determines whether the broker should verify SSL certificates when communicating with the Boot Strap Bosh director"
-    default: true
-  broker.director.url:
+  broker.directors:
     description: "URL of the Bosh director"
```
Comment (kamath-prasad) [inline jobs/service-fabrik-broker/spec]: Update the description.

File: jobs/service-fabrik-scheduler/spec [hunk 3]
```diff
@@ -188,32 +188,8 @@ properties:
...
-  broker.bootstrap_bosh_director.username:
-    description: "Admin username for the Boot Strap Bosh director"
-  broker.bootstrap_bosh_director.password:
-    description: "Admin password for the Boot Strap Bosh director"
-  broker.bootstrap_bosh_director.skip_ssl_validation:
-    description: "Determines whether the broker should verify SSL certificates when communicating with the Boot Strap Bosh director"
-    default: true
-  broker.director.url:
+  broker.directors:
     description: "URL of the Bosh director"
```
Comment (kamath-prasad) [inline jobs/service-fabrik-scheduler/spec]: update the description.

**Comment by kamath-prasad** [comment]
@muralisc : Your branch has conflicts , please resolve and update PR.

File: config/final.yml (hunk 1 above)
Comment (muralisc) [inline config/final.yml]: this is for bosh2 cli
